src/components/DeleteModal.tsx

```diff
@@ -65,7 +65,8 @@ function DeleteModal (props: MyProps & DispatchProps & RouteComponentProps) {
       buttonConfirmName={'Delete'}
       onClickButtonConfirm={handleDelete}
       cancelAction={props.handleClose}
-      disableButtons={busyDeleting}
+      disabledCancelAction={busyDeleting}
+      disabledConfirmAction={busyDeleting}
```

Comment:
I need to separate them into 2 parameters as sometimes the confirm button needs to be disabled but not the cancel action.